src/handlers/payout/post.ts

```diff
@@ -148,8 +154,8 @@ export const calculateIssueCreatorReward = async (incentivesCalculation: Incenti
     incentivesCalculation.paymentPermitMaxPrice
   );
 
-  if (!result || !result.account || !result.amountInETH) {
-    throw new Error("Failed to generate permit for issue creator because of missing account or amountInETH");
+  if (!result || !result.account || !result.rewardInTokens) {
```

Comment:
I don't think that's as expressive as inBigNumber etc it implies 1e18 format

Reply:
it's not in 1e18 format, but with decimals like `5.235`